src/fieldset/index.jsx

```diff
@@ -6,6 +6,17 @@ import classnames from 'classnames';
 import { TextArea, Input, CheckboxGroup, RadioGroup, Select, DateInput } from '@ukhomeoffice/react-components';
 import { Snippet, ConditionalReveal, SpeciesSelector, ApplicationConfirm, RestrictionsField, Markdown, DurationField, SelectMany } from '../';
 
+function Autocomplete(props) {
+  useEffect(() => {
+    const accessibleAutocomplete = require('accessible-autocomplete');
```

Comment:
Does this fail if called outside the `useEffect`?

Reply:
yes, rubbish.